packages/flame/lib/src/components/position_component.dart

```diff
@@ -285,8 +285,9 @@ class PositionComponent extends Component {
   void flipHorizontallyAroundCenter() {
     if (_anchor.x != 0.5) {
       final delta = (1 - 2 * _anchor.x) * width;
```

Comment:
This should be
```
final delta = (1 - 2 * _anchor.x) * width * scale.x;
```
and then the lines below restored as they were before.

Then it would work for components that are rotated + scaled + flipped at the same time.

Reply:
Fixed!